ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/Tests.scala

```diff
@@ -7,12 +7,19 @@ import java.nio.file.Path
 
 import com.daml.ledger.api.testtool.infrastructure.{BenchmarkReporter, Envelope, LedgerTestSuite}
 import com.daml.ledger.api.testtool.suites._
+import com.daml.lf.language.LanguageVersion
+import com.daml.ledger.test.TestDar
 
 import scala.collection.SortedSet
 import scala.concurrent.duration.FiniteDuration
 
 object Tests {
 
+  private val supportsExceptions: Boolean = {
```

Comment:
These are temp code, right? after exceptions table (or preview?) this will be gone, right?

Reply:
Not quite, we build the ledger API test tool for very old LF versions and those are never going to support LF versions. Once we stabilize it this will move to a stable LF version. If we ever stop building the ledger APi test tool for old LF versions, we can kill this completely.

Reply:
Oh, well you know what you do, but going forward like this might increase this type of clutter, so eventually we might need something else. for now it's fine by me.